bindings/c/generate_asm.py

```diff
@@ -118,6 +124,46 @@ def write_unix_asm(asmfile, functions, prefix):
                 assert False, '{} not supported for Arm yet'.format(os)
             asmfile.write("\tldr x8, [x8]\n")
             asmfile.write("\tbr x8\n")
+        elif cpu == "ppc64le":
+            asmfile.write("\n.LCF%d:\n" % (i))
```

Comment:
I'm not familiar with powerpc but this seems like a lot of assembly code to accomplish a tail call to a function pointer. It looks like it might be possible to do this more efficiently: https://godbolt.org/z/onecMa7vh. I checked there that the tail calls for functions with different numbers of arguments generate the same code, so that seems promising (this needs to work for arbitrary function signatures, which is why it's implemented in assembly in the first place)

Reply:
I had tried the same code before but few tests were failing as there are few functions which  receives large number of arguments (i.e., 15 to 17 arguments). 

- fdb_transaction_get_range()
- fdb_transaction_get_range_and_flat_map()

 Hence I have added these instructions to load and process additional parameters.

https://godbolt.org/z/szEKqh74f